cmd/grype/cli/commands/root.go

```diff
@@ -65,7 +65,8 @@ You can also explicitly specify the scheme to use:
     {{.appName}} file:path/to/yourfile                  read directly from a file on disk
     {{.appName}} sbom:path/to/syft.json                 read Syft JSON from path on disk
     {{.appName}} registry:yourrepo/yourimage:tag        pull image directly from a registry (no container runtime required)
-    {{.appName}} purl:path/to/purl/file                 read a newline separated file of purls from a path on disk
+    {{.appName}} purl:path/to/purl/file                 read a newline separated file of package URLs from a path on disk
+    {{.appName}} pkg:PURL                               read a single package PURL directly (e.g. pkg:apk/openssl@3.2.1?distro=alpine-3.20.3)
```

Comment:
this is a doc problem really, since your suggestion is how the software functions (accepts `pkg:...` not `pkg:pkg:...`) but didn't have a good way to nicely say "starts with `pkg` then the rest of the PURL follows"

Reply:
I've updated the docs to say `PURL` over `pkg:PURL`